static/lang/en.json

```diff
@@ -3659,6 +3701,18 @@
         "TempHitPointsShortLabel": "Temp HP",
         "TemplateLabel": "{size}-{unit} {shape}",
         "TemporaryItemToolTip": "This item is temporary and will expire after a duration or during daily preparations. It has no value and cannot be sold.",
+        "Terrain": {
+            "Aquatic": "Aquatic",
+            "Arctic": "Arctic",
+            "Desert": "Desert",
+            "Forest": "Forest",
+            "Mountain": "Mountain",
+            "Plains": "Plains",
+            "Sky": "Sky",
+            "Swamp": "Swamp",
+            "Underground": "Underground",
+            "Urban": "Urban"
+        },
```

Comment:
Can we copy this to `Environment.Type` so at least that part of the move to it will be done?

Reply:
Sure. I'll keep the old values in re-en.json then so that it doesn't pollute the diff.